src/relaycard/frame.py

```diff
@@ -49,5 +48,9 @@ def __init__(self, response):
         self.crc = response[3]
 
     def __repr__(self):
-        return '<ResponseFrame %s/addr:%s data:%s crc:%s>' % (
-            RESPONSES[self.command], self.address, self.data, self.crc)
+        return "<ResponseFrame {}/addr:{} data:{} crc:{}>".format(
```

Comment:
Have you considered migrating to f-strings right away if we touch this piece of code anyway?

Reply:
Done